websites/P/Plex/presence.ts

```diff
@@ -136,25 +136,27 @@ function getTranslation(stringName: string): string {
 let isUploading = false;
 
 const uploadedImages: Record<string, string> = {};
-async function uploadImage(url: string): Promise<string> {
+async function uploadImage(urltoupload: string): Promise<string> {
```

Comment:
For consistent variable naming, something like `url` or `urlToUpload` or `uploadUrl` may be better.

Reply:
i'll apply these tomorrow it was mothers day

Reply:
This doesn't appear to have been resolved.